ci-operator/step-registry/observers/resource-watch/observers-resource-watch-commands.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export REPOSITORY_PATH="${ARTIFACT_DIR}/resource-watch-store"
+
+function cleanup() {
+  echo "killing resource watch"
+  CHILDREN=$(jobs -p)
+  if test -n "${CHILDREN}"
+  then
+    kill ${CHILDREN} && wait
+  fi
+
+  tar -czC $REPOSITORY_PATH -f "${ARTIFACT_DIR}/resource-watch-store.tar.gz" .
```

Comment:
Any idea why this is coming out *not* gzipped? https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/33109/rehearse-33109-periodic-ci-openshift-release-master-ci-4.12-e2e-aws-ovn-serial/1583447789566693376/artifacts/e2e-aws-ovn-serial/observers-resource-watch/artifacts/

Is this the thing we've heard mentioned where gzip appears to be getting automatically undone somewhere in our artifacts stack? (cc @DennisPeriquet @stbenjam, can't recall who mentioned this) If so we should probably just skip the gzip step.

I do have one nit here, could we structure this such that there's another sub-directory in play? I notice when you download the tar and extract it, all the files drop in your current dir. It would be great if they were isolated in a sub-dir so we don't accidentally spew the files all over places we didn't intend to. 

I think this would be using ```REPOSITORY_PATH="${ARTIFACT_DIR}/resource-watch-store/repo"```, but when we go to tar, only using ```${ARTIFACT_DIR}/resource-watch-store/```

Reply:
it looks like it's actually zipped even through it appears with just `.tar`.

```
Mozart:Downloads dperique$ file resource-watch-store.tar 
resource-watch-store.tar: gzip compressed data
Mozart:Downloads dperique$ ls -lh resource-watch-store.tar 
-rw-r--r--@ 1 dperique  staff   3.8M Oct 24 08:59 resource-watch-store.tar

Mozart:Downloads dperique$ mv resource-watch-store.tar resource-watch-store.tar.gz
Mozart:Downloads dperique$ gzip -d resource-watch-store.tar.gz 
Mozart:Downloads dperique$ ls -lh resource-watch-store.tar 
-rw-r--r--  1 dperique  staff    11M Oct 24 08:59 resource-watch-store.tar
```